deepchecks/checks/methodology/boosting_overfit.py

```diff
@@ -113,19 +113,19 @@ class BoostingOverfit(TrainTestBaseCheck):
     """Check for overfit occurring when increasing the number of iterations in boosting models.
 
     The check runs a pred-defined number of steps, and in each step it limits the boosting model to use up to X
-    estimators (number of estimators is monotonic increasing). It plots the given metric calculated for each step for
+    estimators (number of estimators is monotonic increasing). It plots the given score calculated for each step for
     both the train dataset and the test dataset.
 
     Args:
-        metric (Union[Callable, str]): Metric to use verify the model, either function or sklearn scorer name.
-        metric_name (str): Name to be displayed in the plot on y-axis. must be used together with 'metric'
+        scorer (Union[Callable, str]): Scorer to use verify the model, either function or sklearn scorer name.
```

Comment:
```suggestion
        scorer (Union[Callable, str]): Scorer used to verify the model, either function or sklearn scorer name.
```

Reply:
and a question - @noamzbr how is it possible with a function? isn't it a scorer object? https://scikit-learn.org/stable/modules/model_evaluation.html#defining-your-scoring-strategy-from-metric-functions

Reply:
A scoring object is a function (Callable), they say object but mean a Callable (they even state it explicitly later on)

Reply:
👍